packages/plugins/users-permissions/server/config.js

```diff
@@ -11,19 +19,14 @@ module.exports = {
      * - "legacy-support": use plugin JWTs (backward compatible)
      * - "refresh": use SessionManager (access/refresh tokens)
      */
-    jwtManagement: env('UP_JWT_MANAGEMENT', 'legacy-support'),
+    jwtManagement: 'legacy-support',
     sessions: {
-      accessTokenLifespan: env.int('UP_SESSIONS_ACCESS_TTL', 7 * 24 * 60 * 60), // 1 week
-      maxRefreshTokenLifespan: env.int('UP_SESSIONS_MAX_REFRESH_TTL', 30 * 24 * 60 * 60), // 30 days
-      idleRefreshTokenLifespan: env.int('UP_SESSIONS_IDLE_REFRESH_TTL', 7 * 24 * 60 * 60), // 7 days
-      httpOnly: env.bool('UP_SESSIONS_HTTPONLY', false),
-      cookie: {
-        name: env('UP_SESSIONS_COOKIE_NAME', 'strapi_up_refresh'),
-        sameSite: env('UP_SESSIONS_COOKIE_SAMESITE', 'lax'),
-        path: env('UP_SESSIONS_COOKIE_PATH', '/'),
-        domain: env('UP_SESSIONS_COOKIE_DOMAIN'),
-        secure: env.bool('UP_SESSIONS_COOKIE_SECURE', process.env.NODE_ENV === 'production'),
-      },
+      accessTokenLifespan: DEFAULT_ACCESS_TOKEN_LIFESPAN,
+      maxRefreshTokenLifespan: DEFAULT_MAX_REFRESH_TOKEN_LIFESPAN,
+      idleRefreshTokenLifespan: DEFAULT_IDLE_REFRESH_TOKEN_LIFESPAN,
+      maxSessionLifespan: DEFAULT_MAX_SESSION_LIFESPAN,
```

Comment:
I removed these because we don't want to add new hardcoded ENV var in strapi code. It should be up to the user to set it in their config.